gschem/src/x_newtext.c

```diff
@@ -75,7 +75,8 @@ dialog_response_apply (NewText *dialog)
   int align = LOWER_LEFT;
   int color = TEXT_COLOR;
   int rotate = 0;
-  int size = 12;
+  GschemToplevel *w_current = GSCHEM_DIALOG (dialog)->w_current;
```

Comment:
Agree, it's better to use properties. Then, `x_multiattrib.c`, `x_pagesel.c` and `x_compselect.c` should be fixed in the same manner?